spec/factories/users.rb

```diff
@@ -2,7 +2,12 @@
 
 FactoryBot.define do
   factory :user do
-    email { Faker::Internet.email }
+    email do
+      generated_email = Faker::Internet.email
+      generated_email.sub!('.example', '.com')
+      generated_email.sub!('.test', '.com')
+      generated_email
+    end
```

Comment:
You could do something like:
```suggestion
    email { Faker::Internet.email(domain: 'email.com') }
```
to guarantee that there are no other extensions aside from example or test